tools/reviewer/web/src/app/dashboard/reviews-panel/review/person-list/person-list.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Diff,
+  FirebaseService,
+  NotificationService,
+  ProtoService,
+  Status
+} from '@/shared';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-person-list',
+  templateUrl: './person-list.component.html',
+  styleUrls: ['./person-list.component.scss']
+})
+export class PersonListComponent {
+  @Input() diff: Diff;
+  @Input() property: string;
```

Comment:
What kind of property is this?
If it's the name list, let's call it that. Also, if we assume it's comma-delimited, let's write it here too.

Reply:
property can be 'reviewers'  or 'cc'. It is the Diff property name of what we are showing. reviewers or ccs